cmd/gntagger/main.go

```diff
@@ -47,10 +52,17 @@ func main() {
 			os.Exit(0)
 		}
 	case 1:
-		path = flag.Arg(0)
-		data, err = ioutil.ReadFile(path)
-		if err != nil {
-			log.Panic(err)
+		command := flag.Arg(0)
+		switch command {
+		case "version":
```

Comment:
we use flag pkg, to it makes sense to add --verion/-v flag for this

Reply:
[`flag.Bool`](https://github.com/gnames/gntagger/pull/42/files#diff-885fc44ff6e3489ef27c48611e8deba7R45) doesn't have several options for the same flag. Should `gntagger` have several definitions? It would affect `flag.PrintDefaults()`.